user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md

```diff
@@ -4,6 +4,8 @@ uid: Working_with_pattern_matching
 
 # Working with pattern matching
 
+## Univariate patterns
+
 From DataMiner 10.0.7 onwards, on systems using a Cassandra and Elasticsearch database, DataMiner Analytics can automatically recognize recurring patterns in trend data. For this purpose, you first need to [define the patterns](xref:Defining_a_pattern) DataMiner Analytics needs to track. This pattern definition is also known as a "tag".
```

Comment:
wondering if this "Defining a pattern" page also requires a little change as it starts with "In a trend graph showing trend information for one single parameter....": you can now also do it a trend graph showing trend information for multiple parameters, in which case a multivariate pattern will be created

Reply:
Implemented in [commit 956ed0b](https://github.com/SkylineCommunications/dataminer-docs/pull/1699/commits/956ed0bb38e2673a5608f0c9fd379841b7732675).